corehq/apps/domain/forms.py

```diff
@@ -498,7 +498,7 @@ def save(self, request, domain):
         domain.hr_name = self.cleaned_data['hr_name']
         domain.project_description = self.cleaned_data['project_description']
         domain.default_mobile_ucr_sync_interval = self.cleaned_data.get('mobile_ucr_sync_interval', None)
-        domain.default_geocoder_location = self.cleaned_data.get('default_geocoder_location')
+        domain.default_geocoder_location = json.loads(self.cleaned_data.get('default_geocoder_location','{}'))
```

Comment:
Nitpick: I'd do this in the `clean_default_geocoder_location` method. That way if the value is used in multiple places, the json decode only needs to be done once.